### DISCUSSION TITLE
Why does `ls` show the wrong directory size?

### DISCUSSION THREAD

**Discussion by txtyash** [nushell/nushell#5815] [Q&A]
When using `ls` without the `-d` flag, `ls` shows the wrong directory size.

**EDIT:**
Using `--du` flag every time solves my problem.

**Comment by kubouch** [comment]
It's working as expected, check here: https://askubuntu.com/questions/186813/why-does-every-directory-have-a-size-4096-bytes-4-k

Recursing every time with `--du` by default would be too expensive.
